packages/labextension/style/index.css

```diff
@@ -73,17 +71,17 @@
   vertical-align: top;
 }
 
-
-
 /* Hiding unchanged cells if told to */
 .nbdime-root.jp-mod-hideUnchanged .jp-Cell-diff.jp-Diff-unchanged {
   display: none;
 }
 
 /* Show a marker with the number of cells hidden before */
-.nbdime-root.jp-mod-hideUnchanged .jp-Cell-diff[data-nbdime-NCellsHiddenBefore]::before,
-.nbdime-root.jp-mod-hideUnchanged .jp-Diff-addremchunk[data-nbdime-NCellsHiddenBefore]::before {
-  content: attr(data-nbdime-NCellsHiddenBefore) " unchanged cell(s) hidden";
+.nbdime-root.jp-mod-hideUnchanged
+  .jp-Cell-diff[data-nbdime-NCellsHiddenBefore]::before,
```

Comment:
I would prefer it if single CSS rules weren't split across lines (if such a rule config is easily available.

Reply:
I have tried to add `--print-width=120` in prettier script option in the package.json file but it doesn't seem to produce effect on the width of the lines.